doc/manual/manual.tex

```diff
@@ -524,8 +524,10 @@ \subsubsection{A comment on adiabatic heating}
   & \approx -\alpha \rho T \mathbf u \cdot \mathbf g.
 \end{align*}
 While this simplification is possible, it is not necessary if you have access
-to the total pressure. \aspect{} therefore implements the original term
-without this simplification.
+to the total pressure. \aspect{} therefore by default implements the original 
+term without this simplification, but allows to simplify this term by setting
+the \textit{Use simplified adiabatic heating} 
```

Comment:
We seem to consistently use `\texttt` for parameter names. 

Would it also be worthwhile adding a link to the corresponding section of this parameter?